static/app/bootstrap/exportGlobals.tsx

```diff
@@ -63,11 +49,8 @@ globals.SentryApp = {
   },
 
   // The following components are used in legacy django HTML views
-  passwordStrength: {load: loadPasswordStrength},
   U2fSign: require('app/components/u2f/u2fsign').default,
   ConfigStore: require('app/stores/configStore').default,
-  SystemAlerts: require('app/views/app/systemAlerts').default,
-  Indicators: require('app/components/indicators').default,
```

Comment:
Some/almost all of these other globals could be removed if they are not used by legacy plugins

Reply:
I think most of these are still used unfortunately. U2FSign for U2FA during login, SetupWizard is for the sdk setup wizard, HookStore is for the extra demo js, Modal is for a few places we use the modal in django templates. Not sure about Forms, LoadingIndicator and ConfigStore tho

Reply:
Re: `U2FSign` and `SetupWizard`, they can get the same treatment as the other components in this PR (I opted not to include them as my primary purpose was addressing templates that used jQuery).

Good to know re: `HookStore`, forgot about that -- @scefali should we include a simple test that makes sure `HookStore` is exposed as a global so that someone doesn't accidentally remove it?